index.d.ts

```diff
@@ -252,10 +252,20 @@ declare namespace execa {
 
 	interface ExecaReturnBase<StdoutStderrType> {
 		/**
-		The file and arguments that were run.
+		The file and arguments that were run, for logging purpose.
+
+		This is not escaped and should be passed to neither `execa()` nor `execa.command()`
 		*/
 		command: string;
 
+		/**
+		Same as `command` but escaped.
+
+		This is meant to be copy/pasted in a shell, for debugging purpose.
+		Since the escaping is fairly basic, this should be passed to neither `execa()` nor `execa.command()`
```

Comment:
To prevent any vulnerability report coming from this, I think we should say that it should not be used with any child process type method. Same with `.command`.

Reply:
Fixed :+1: